packages/swirl-components/src/components/swirl-popover-trigger/swirl-popover-trigger.tsx

```diff
@@ -71,35 +72,48 @@ export class SwirlPopoverTrigger {
     }
   }
 
-  private onClick = () => {
-    const popoverEl = this.getPopoverEl();
+  private onMouseenter = () => {
```

Comment:
Issue: I think opening and closing the popover via hover should not move focus. You could extend the SwirlPopover open and close methods to take an optional param for disabling the focus changes.

Reply:
My initial idea was to minimize code but I agree that it is harder to reason about.

Done

Reply:
I am not sure how this functionality works but I added the optional param and added it to checks to prevent focus logic